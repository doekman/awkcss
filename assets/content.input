1. First line
2. Second line
3. Third line
4. Fourth line
