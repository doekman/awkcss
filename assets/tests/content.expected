0. The super header selector (::before)      
1. First line                                
2. Second line                               
3. Third line                                
4. Fourth line                               
9. The super footer selector (::after)       
